examples/chip-tool/templates/tests.js

```diff
@@ -15,6 +15,29 @@
  *    limitations under the License.
  */
 
+function getManualTests()
+{
+  const DeviceDiscovery = [
+    'Test_TC_DD_1_5',
+    'Test_TC_DD_1_6',
+    'Test_TC_DD_1_7',
+    'Test_TC_DD_1_8',
+    'Test_TC_DD_1_9',
+  ];
+
+  const Groups = [
+    'TestGroupDemoCommand',
```

Comment:
Fwiw, as a followup we can remove the manual exclusion of these in `.github/workflows/tests.yaml`.

Reply:
Ok. I created an issue [here](https://github.com/project-chip/connectedhomeip/issues/14753)